1_densenet.ipynb

```diff
@@ -117,7 +117,7 @@
     "test = datasets.ImageFolder(data_dir+'/val', transform=test_transforms)\n",
     "\n",
     "trainloader = torch.utils.data.DataLoader(train, batch_size=16, shuffle=True, num_workers=4)\n",
-    "testloader = torch.utils.data.DataLoader(test, batch_size=8, shuffle=True, num_workers=4)"
+    "testloader = torch.utils.data.DataLoader(test, batch_size=16, shuffle=True, num_workers=4)"
```

Comment:
@devon12stone I changed `batch_size` here because the batch size of the testloader actually doesn't affect performance since there is no backpropagation. Increasing batch size speeds up training and inference, but too large and you can exceed your computer's memory. I have also noticed in the past that an increased batch size leads to a smoother training loss curve, but too large can flatten the curve out. It's almost like a bias-variance tradeoff scenario